main.js

```diff
@@ -1,15 +1,25 @@
 #!/usr/bin/env node
 
 const {restorePackageJson} = require("./restore-package-json.function");
+const {restoreOriginalPackageJson} = require("./restore-original-package-json.function");
 const {exec} = require('child_process');
 const {promises} = require('fs');
 const {join} = require('path');
 
+let packagePaths;
+
 process.on('unhandledRejection', (error) => {
     console.error(error);
```

Comment:
I'm thinking about if it would also be useful to gracefully "die" at this point, but I see the "problem" that there isn't necessarily something to restore 🤔

Reply:
Yes, you are right. The current process fills `packagePaths` "fast enough" so that such an situation won't occur. But I agree that this is something to consider. I'll ensure that `restoreOriginalPackageJson` is able to handle that. :+1: 
Thx a lot for the hint! =)